gravitee-management-api-service/src/main/java/io/gravitee/management/service/impl/ApiServiceImpl.java

```diff
@@ -556,8 +564,30 @@ public String convertAsJsonForExport(final String apiId) {
         apiEntity.setDeployedAt(null);
         apiEntity.setPrimaryOwner(null);
         apiEntity.setState(null);
+
```

Comment:
Why the method name is not simply "_exportAsJson_" instead of "_convertAsJsonForExport_" ?